### PR TITLE
add docs for json default values

### PR DESCRIPTION
**Pull request by nikolasburk**

### CODE REVIEW

File: content/200-orm/100-prisma-schema/20-data-model/10-models.mdx [hunk 1]
```diff
@@ -795,6 +797,7 @@ Default values can be:
 - Static values that correspond to the field type, such as `5` (`Int`), `Hello` (`String`), or `false` (`Boolean`)
 - [Lists](/orm/reference/prisma-schema-reference#-modifier) of static values, such as `[5, 6, 8]` (`Int[]`) or `["Hello", "Goodbye"]` (`String`[]). These are available in Prisma ORM versions `4.0.0` and later, when using supported databases (PostgreSQL, CockroachDB and MongoDB)
 - [Functions](#using-functions), such as [`now()`](/orm/reference/prisma-schema-reference#now) or [`uuid()`](/orm/reference/prisma-schema-reference#uuid)
+- JSON data which needs to be enclosed with double-quotes inside the `@default` attribute, e.g.: `@default("[]")`. If you want to provide a JSON object, you need to escape its double quotes using a backslash, e.g.: `@default("{ \"hello\": \"world\" }")`.
```
Comment (jharrell) [inline content/200-orm/100-prisma-schema/20-data-model/10-models.mdx]:
```suggestion
- JSON data. Note that JSON needs to be enclosed with double-quotes inside the `@default` attribute, e.g.: `@default("[]")`. If you want to provide a JSON object, you need to enclose it with double-quotes and then escape any internal double quotes using a backslash, e.g.: `@default("{ \"hello\": \"world\" }")`.
```

File: content/200-orm/200-prisma-client/200-special-fields-and-types/100-working-with-json-fields.mdx [hunk 2]
```diff
@@ -1000,3 +1000,15 @@ No - it is not yet possible to filter on the presence of a specific key.
 ### Is case insensitive filtering supported?
 
 No - [case insensitive filtering](https://github.com/prisma/prisma/issues/7390) is not yet supported.
+
+### How to set a default value for JSON fields?
+
+When you want to set a `@default` value the `Json` type, you need to enclose it with double-quotes inside the `@default` attribute, for example:
```
Comment (jharrell) [inline content/200-orm/200-prisma-client/200-special-fields-and-types/100-working-with-json-fields.mdx]: I'd also mention the escaping thing here.

File: content/200-orm/500-reference/100-prisma-schema-reference.mdx [hunk 3]
```diff
@@ -1605,10 +1605,10 @@ Defines a [default value for a field](/orm/prisma-schema/data-model/models#defin
   - [`cuid()`](#cuid)
   - [`uuid()`](#uuid)
   - [`now()`](#now)
-
 - Default values that cannot yet be represented in the Prisma schema are represented by the [`dbgenerated(...)` function](#dbgenerated) when you use [introspection](/orm/prisma-schema/introspection).
 - Default values are not allowed on relation fields in the Prisma schema. Note however that you can still define default values on the fields backing a relation (the ones listed in the `fields` argument in the `@relation` attribute). A default value on the field backing a relation will mean that relation is populated automatically for you.
 - Default values can be used with [scalar lists](/orm/prisma-client/special-fields-and-types/working-with-scalar-lists-arrays) in databases that natively support them.
+- JSON data which needs to be enclosed with double-quotes inside the `@default` attribute, e.g.: `@default("[]")`. If you want to provide a JSON object, you need to escape its double quotes using a backslash, e.g.: `@default("{ \"hello\": \"world\" }")`.
```
Comment (jharrell) [inline content/200-orm/500-reference/100-prisma-schema-reference.mdx]: I'd recommend my previous suggestion here as well.

```suggestion
- JSON data. Note that JSON needs to be enclosed with double-quotes inside the `@default` attribute, e.g.: `@default("[]")`. If you want to provide a JSON object, you need to enclose it with double-quotes and then escape any internal double quotes using a backslash, e.g.: `@default("{ \"hello\": \"world\" }")`.
```

File: content/200-orm/200-prisma-client/200-special-fields-and-types/100-working-with-json-fields.mdx (hunk 2 above)
Comment (nikolasburk) [reply content/200-orm/200-prisma-client/200-special-fields-and-types/100-working-with-json-fields.mdx]:
```suggestion
When you want to set a `@default` value the `Json` type, you need to enclose it with double-quotes inside the `@default` attribute (and potentially escape any "inner" double-quotes using a backslash), for example:
```

**Review by mhessdev — APPROVED** [review]

**Pull request merged**